hw/vendor/pulp_platform_apb.vendor.hjson

```diff
@@ -0,0 +1,17 @@
+// Copyright 2020 ETH Zurich and University of Bologna.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  name: "pulp_platform_apb",
+  target_dir: "pulp_platform_apb",
+
+  upstream: {
+    url: "https://github.com/pulp-platform/apb.git",
+    rev: "master",
+  },
+
+  patch_dir: "patches/pulp_platform_apb",
+
+  exclude_from_upstream: [
+  ]
```

Comment:
Perhaps we can/should exclude some of the files?

Reply:
Maybe the `scripts` folder?